pkg/actuators/machine/loadbalancers.go

```diff
@@ -154,3 +167,24 @@ func gatherLoadBalancerTargetGroups(client awsclient.Client, names []string) ([]
 
 	return targetGroups, nil
 }
+
+// gatherLoadBalancerTargetGroupRegisteredTargets looks for all targets that are registered to a particular targetGroup.
+// Within the GCP API, the only way to find the targets that are registered is to look at the target health for the group.
```

Comment:
GCP?

Reply:
🤦  I was dealing with load balancers in GCP earlier in the day when I typed this